server/handler.js

```diff
@@ -20,3 +34,33 @@ module.exports.hello = async event => {
   // Use this code if you don't use the http event with the LAMBDA-PROXY integration
   // return { message: 'Go Serverless v1.0! Your function executed successfully!', event };
 };
+
+module.exports.createUser = async event => {
```

Comment:
eventually we can move all the endpoint functions out of this generic handler into other files, otherwise this will end up being a pretty massive file. For example we can make a user file that holds functionality for creating and updating users.

Reply:
Sure